slackgpt/ChatBotThread.py

```diff
@@ -16,8 +17,8 @@ def __init__(self, handler: Handler, browser: str, prefix: str = "!", headless:
             headless (bool, optional): Whether to run ChatGPT in headless mode. Defaults to True.
         """
         super().__init__()
-        self.queue = handler.queue
-        self.handler = handler
+        self.queue: Queue = handler.queue
```

Comment:
It would be really nice if we could get rid of this reference here - or maybe need to refactor some major code parts, because it suggests that single-responsibility principle is violated. If we ever wanted to change the inner workings of handler we would potentially have to rework the code here as well